projects/storefrontlib/src/cms-components/checkout/services/checkout-details.service.ts

```diff
@@ -32,6 +33,7 @@ export class CheckoutDetailsService {
     private cartService: CartService
   ) {
     this.cartId$ = this.cartService.getActive().pipe(
+      distinctUntilChanged(),
```

Comment:
better move this `distinctUntilChanged` after `filter`. Check 'cartId' not 'cart content'.